src/clue/components/clue-app-header.tsx

```diff
@@ -175,6 +176,29 @@ export const ClueAppHeaderComponent: React.FC<IProps> = observer(function ClueAp
     );
   };
 
+  const renderProblemInfo = () => {
+    // Only show the problem menu if the user is in standalone mode as we currently
+    // only support switching in standalone for students.  Teachers should already
+    // see the class switcher via the renderNonStudentHeader function unless the
+    // showClassSwitcher flag is set to false in which case they will get here
+    // as the default case (in which case we still don't show the problem menu since
+    // the flag is still false).
+    if ((ui.standalone || user.standaloneAuthUser) && appConfig.showClassSwitcher) {
+      return (
+        <div className="problem-dropdown" data-test="user-problem">
```

Comment:
NOTE: this else is covered by a Cypress test  - not sure why this is being flagged.

https://github.com/concord-consortium/collaborative-learning/blob/master/cypress/e2e/functional/standalone_tests/standalone_load_spec.js#L83